src/test/java/core/basesyntax/file/FileWriterImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.file;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileWriterImplTest {
+    private static final String TEST_FILE_PATH = "test_output.txt";
+    private FileWriterImpl fileWriter;
+
+    @BeforeEach
+    void setUp() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    void write_validData_success() throws IOException {
+        String testData = "Hello, world!";
+
+        fileWriter.write(testData, TEST_FILE_PATH);
+
+        String fileContent = Files.readString(Path.of(TEST_FILE_PATH));
+        assertEquals(testData, fileContent, "File content should match the written data");
+
+        Files.deleteIfExists(Path.of(TEST_FILE_PATH));
+    }
+
+    @Test
+    void write_invalidPath_throwsRuntimeException() {
+        String invalidPath = "/invalid_path/test.txt";
```

Comment:
You can move it to the constants, it will be clearer that you have a test with such a check